front/app/containers/Admin/projects/project/general/index.tsx

```diff
@@ -290,24 +310,24 @@ const AdminProjectsProjectGeneral = () => {
           }
         }
 
-        const imagesToAddPromises = projectImages
-          .filter((file) => !file.remote)
-          .map((file) => {
-            if (latestProjectId) {
-              return addProjectImage(latestProjectId, file.base64);
-            }
+        const cardImageWasChanged =
+          croppedProjectCardBase64 &&
+          projectCardImage &&
+          !projectCardImage.remote;
+        const cardImageToAddPromise =
+          cardImageWasChanged && latestProjectId
+            ? addProjectImage(latestProjectId, croppedProjectCardBase64)
+            : null;
+
+        const cardImageToRemovePromise =
+          projectCardImageToRemove &&
+          projectCardImageToRemove.remote &&
+          projectCardImageToRemove.id &&
+          isString(projectCardImageToRemove.id) &&
+          latestProjectId
+            ? deleteProjectImage(latestProjectId, projectCardImageToRemove.id)
+            : null;
```

Comment:
How about this?

```
const cardImageToRemovePromise =
          projectCardImageToRemove?.id && latestProjectId
            ? deleteProjectImage(latestProjectId, projectCardImageToRemove.id)
            : null;
```

You don't need the remote check normally.

Reply:
I didn't want to remove the previously existing checks, but maybe it can be done.

Reply:
I can't think of why we would need the remote check. We btw already check for it when we set the `projectCardImageToRemove`.

```
    projectCardImageToRemove.remote &&
      setProjectCardImageToRemove(projectCardImageToRemove);
```